jbpm-form-modeler-field-types/jbpm-form-modeler-document/src/main/java/org/jbpm/formModeler/fieldTypes/document/handling/JBPMDocumentFieldTypeHandler.java

```diff
@@ -151,14 +150,14 @@ public Object getValue(Field field, String inputName, Map parametersMap, Map fil
 
     protected AbstractDocumentMarshallingStrategy getDocumentMarshallingStrategy( String inputName ) {
         FormRenderContext context = formRenderContextManager.getRootContext( inputName );
-        DeployedUnit deployedUnit = deploymentService.getDeployedUnit(context.getDeploymentId());
-        InternalRuntimeManager manager = (InternalRuntimeManager) deployedUnit.getRuntimeManager();
-        ObjectMarshallingStrategy[] strategies = (ObjectMarshallingStrategy[]) manager.getEnvironment().getEnvironment().get( EnvironmentName.OBJECT_MARSHALLING_STRATEGIES );
-        if (strategies != null) {
-            for (ObjectMarshallingStrategy strategy : strategies) {
-                if (strategy instanceof AbstractDocumentMarshallingStrategy) return (AbstractDocumentMarshallingStrategy) strategy;
-            }
-        }
+//        DeployedUnit deployedUnit = deploymentService.getDeployedUnit(context.getDeploymentId());
```

Comment:
What about this commented code? Seeing `InternalRuntimeManager` i think this should just be removed.